src/main/java/com/Acrobot/ChestShop/Listeners/Modules/PriceRestrictionModule.java

```diff
@@ -17,14 +15,27 @@
 import org.bukkit.event.Listener;
 import org.bukkit.inventory.ItemStack;
 
-import java.io.File;
-import java.io.IOException;
-import java.math.BigDecimal;
-import java.util.Locale;
-import java.util.logging.Level;
-
-import static com.Acrobot.ChestShop.Events.PreShopCreationEvent.CreationOutcome.*;
-import static com.Acrobot.ChestShop.Permission.*;
+import com.Acrobot.Breeze.Utils.PriceUtil;
+import com.Acrobot.ChestShop.ChestShop;
+import com.Acrobot.ChestShop.Configuration.Messages;
+import com.Acrobot.ChestShop.Events.ChestShopReloadEvent;
+import com.Acrobot.ChestShop.Events.ItemParseEvent;
+import com.Acrobot.ChestShop.Events.PreShopCreationEvent;
+import static com.Acrobot.ChestShop.Events.PreShopCreationEvent.CreationOutcome.BUY_PRICE_ABOVE_MAX;
+import static com.Acrobot.ChestShop.Events.PreShopCreationEvent.CreationOutcome.BUY_PRICE_BELOW_MIN;
+import static com.Acrobot.ChestShop.Events.PreShopCreationEvent.CreationOutcome.SELL_PRICE_ABOVE_MAX;
+import static com.Acrobot.ChestShop.Events.PreShopCreationEvent.CreationOutcome.SELL_PRICE_BELOW_MIN;
+import com.Acrobot.ChestShop.Permission;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MAX_BUY;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MAX_BUY_ID;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MAX_SELL;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MAX_SELL_ID;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MIN_BUY;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MIN_BUY_ID;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MIN_SELL;
+import static com.Acrobot.ChestShop.Permission.NOLIMIT_MIN_SELL_ID;
+import com.Acrobot.ChestShop.Signs.ChestShopSign;
+import com.Acrobot.ChestShop.Utils.ItemUtil;
```

Comment:
These seem like a lot of line changes which I'm unsure they are all required. I assume your IDE did that automatically but I would prefer it if you'd only included changed imports that are actually required by the implemented function to keep the git history consistent. (I realise that the plugin does not have one consistent code style, adjusting that is on my todo but should be done globally imo)

Reply:
Changed - will let you mark as resolved on evaluation